proposals/20210917-mitre-tags.md

```diff
@@ -0,0 +1,12 @@
+# Add missing MITRE ATT&CK tags
+
+## Summary
+This is a proposal for adding missing MITRE ATT&CK tags to the existing Falco rules.
+
+We put a spreasheet together with our proposal at https://docs.google.com/spreadsheets/d/1lK2luf4iuOHDJEZ9EGcv3DkuuHeyRJAfeBzQCu2AuUI that we opened up for discussion during the community calls ([1](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-09), [2](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-23), [3](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-14), [4](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-21)).
```

Comment:
Thank you @seishinsha for your effort 🙇‍♂️ 

Anyways, who owns this spreadsheet?
I don't want to be pedantic, but I'd prefer to have the info contained by the spreadsheet here in the Falco repo so that the community can benefit at any moment, especially in the future even if that spreadsheet disappears or gets lost.

It's for the same reasons, we store community notes in falcosecurity/community, Falco proposals in the /proposals directory etc.

BTW, I'd vote +1 to move such documents to GitHub discussions.